wasm-pack.yaml

```
@@ -0,0 +1,158 @@
+package:
+  name: wasm-pack
+  version: 0.12.1
+  epoch: 0
+  description: rust to wasm build tool
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - build-base
+      - busybox
+      - bzip2-dev
+      - ca-certificates-bundle
+      - cargo-auditable
+      - curl-dev
+      - openssl-dev
+      - rust
+      - zstd-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/rustwasm/wasm-pack
+      expected-commit: e25150ecd322fc03c9abcd141bc979b108e8a344
+      tag: v${{package.version}}
+
+  - uses: patch
+    with:
+      patches: upgrade-deps.patch
+
+  - runs: |
+      # prepare
+      rm -rf Cargo.lock
+      cargo fetch --target="${{host.triplet.rust}}"
+
+      cargo auditable build --release --frozen
+      install -Dm755 target/release/wasm-pack \
+      -t "${{targets.contextdir}}"/usr/bin
+
+  - uses: strip
+
+update:
+  enabled: true
+  github:
+    identifier: rustwasm/wasm-pack
+    strip-prefix: v
+
+test:
+  environment:
+    contents:
+      packages:
+        - nodejs
+        - npm
+        - rustup
+        - yarn
+        - build-base
+        - binaryen
+  pipeline:
+    - runs: |
+        # Function to display messages
+        function print_info {
+            echo -e "\033[1;34m[INFO]\033[0m $1"
+        }
+
+        function print_error {
+            echo -e "\033[1;31m[ERROR]\033[0m $1"
```

```suggestion
            echo -e "\033[1;31m[ERROR]\033[0m $1"
            exit 1
```

Everywhere `print_error` is called, we call `exit 1` after, we can just do that in here and save some lines.